scripts/helpers/downloads.js

```diff
@@ -139,11 +139,19 @@ function resolveUrl (item, version) {
 
 module.exports = (version) => {
   let downloads = latestDownloads
-  if (semver.satisfies(version, '< 8.0.0')) {
-    downloads = postMergeDownloads
-  }
   if (semver.satisfies(version, '< 1.0.0')) {
     downloads = legacyDownloads
+  } else if (semver.satisfies(version, '< 8.0.0')) {
+    downloads = postMergeDownloads
+  } else if (semver.satisfies(version, '>= 10.0.0')) {
+    downloads = postMergeDownloads.map(value => {
+      if (value.title === 'Linux 32-bit Binary' ||
+          value.title === 'SunOS 32-bit Binary') {
+        // Indicate that comingSoon shouldn't be shown
+        value.notShowComingSoon = true
```

Comment:
Nit: wouldn't it be easier to use `showComingSoon` instead of it's negation? Then below we can do

```js
if (binary.showComingSoon) {
  return `${binary.title}: *Coming soon*`
}
return binary.title
```

It's the same but I personally find it easier to grok.

Reply:
The reason is that we DON'T wanna show commingSoon, and only those versions >= 10.0.0 will meet the requirement. So most of downloadable items won't have this symbol (They are shown in default, ONLY with this symbol won't be shown).

If I change to yours, I'll have to add the property to the original `postMergeDownloads` JSON objects. In fact there's no need doing that. Hope you understand :)

Reply:
Yes, I do.

Reply:
Perhaps you can use something like `hideComingSoon` as property name then.

Reply:
OK. Thanks!
And we're still considering whether there's anything better or easier way……Maybe my 1st way is the best, or am I thinking of it a bit too complicated? :D